ccd-definition/CaseEventToFields/DefendantResponseAdmitPartLRspec.json

```diff
@@ -43,8 +43,8 @@
     "PageFieldDisplayOrder": 3,
     "DisplayContext": "MANDATORY",
     "PageID": "defenceAdmittedPartRoute",
-    "ShowSummaryChangeOption": "Y",
-    "FieldShowCondition": "respondent1ClaimResponseTypeForSpec=\"PART_ADMISSION\" AND specDefenceAdmittedRequired = \"Yes\"",
```

Comment:
When we activated the full admission journey there was a problem with backtracking: lets say the user chooses admits part, selects specDefenceAdmittedRequired = Yes, then, maybe a couple screens after that, goes back, and chooses admits full. specDefenceAdmittedRequired is still yes, so this field will appear. Have you checked that there is no such backtracking problems before removing the =PART_ADMISSION part?